virttest/tests/unattended_install.py

```diff
@@ -1157,6 +1157,38 @@ def string_in_serial_log(serial_log_file_path, string):
         return False
 
 
+def attempt_to_log_useful_files(test, vm):
+    """
+    Uses "vm.serial_console" to look in the system for useful log files
+    """
+    console = vm.serial_console
+    for path_glob in ["/*.log", "/tmp/*.log", "/var/tmp/*.log"]:
+        try:
+            paths = console.cmd("ls -1 %s" % path_glob)
+        except Exception as details:
+            continue
+        for path in paths.splitlines():
+            if not path:
+                continue
+            if path.startswith(os.path.sep):
+                rel_path = path[1:]
+            else:
```

Comment:
How wouldn't the path not start with os.path.sep?

Reply:
In case someone extends the current list: `["/*.log", "/tmp/*.log", "/var/tmp/*.log"]` or some obscure system returns the "ls" output with relative links :-). Anyway if you insist I can remove that...

Reply:
Given the current `path_glob` you're right.  But, on the other hand, it's a nice extra check, given that any absolute path on `os.path.join` will take precedence on (and remove) the previous paths.